lib/dotcom_web/live/subway_alerts.ex

```diff
@@ -0,0 +1,159 @@
+defmodule DotcomWeb.Live.SubwayAlerts do
+  @moduledoc """
+  The subway alerts/status page, showing current subway status, planned work, and
+  other upcoming and relevant alerts.
+  """
+
+  use DotcomWeb, :live_view
+
+  import DotcomWeb.Components.PlannedDisruptions, only: [disruptions: 1]
+  import DotcomWeb.Components.TAlerts, only: [t_alerts: 1]
+  import DotcomWeb.Components.SystemStatus.SubwayStatus, only: [alerts_subway_status: 1]
+
+  alias Alerts.Alert
+  alias Alerts.InformedEntity
+  alias Alerts.Match
+  alias Dotcom.Alerts.Subway.Disruptions
+
+  @alerts_repo Application.compile_env!(:dotcom, :repo_modules)[:alerts]
+  @routes_repo Application.compile_env!(:dotcom, :repo_modules)[:routes]
+
+  @date_time Application.compile_env!(:dotcom, :date_time_module)
+
+  embed_templates "layouts/*"
+
+  def mount(_params, _session, socket) do
+    {:ok,
+     socket
+     |> assign(
+       :meta_description,
+       "Live service alerts for all MBTA transportation modes, including subway, bus, Commuter Rail, and ferry. " <>
+         "Updates on delays, construction, elevator outages, and more."
+     )
+     |> assign(:breadcrumbs, [Breadcrumb.build("Alerts")])
+     |> assign_now()
+     |> assign_route_type()
+     |> assign_routes()
+     |> assign(:subway_status, Dotcom.SystemStatus.subway_status())
+     |> assign(:disruption_groups, Disruptions.future_disruptions())
+     |> assign_alerts()
+     |> assign_banner_alert()
+     |> assign_alert_groups()}
```

Comment:
Do we need _all_ of these as separate steps in the pipeline here? I'd recommended either consolidating the ones that are dependent on each other, or extract ones that'd be useful in other views into a separate [on_mount hook](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.html#on_mount/1) (or perhaps some combination of both).